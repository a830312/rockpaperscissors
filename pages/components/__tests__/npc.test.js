/* eslint-env jest */

import React from "react"
import renderer from "react-test-renderer"
import App from "../npc.js"

describe("with snapshot testing", () => {
  it("matches the snapshot", () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})