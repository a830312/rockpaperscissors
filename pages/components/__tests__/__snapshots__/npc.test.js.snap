// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`with snapshot testing matches the snapshot 1`] = `
<p>
  NPC
  
   choose 
  <strong>
    {weapon}
  </strong>
   and 
  <strong>
    {{win / loss}}
  </strong>
</p>
`;
